### PR TITLE
Clarify allowed example submission location

### LINKED ISSUE #306: Unclear requirement for placement of example submissions
In section "Example Submissions" we say

> Submissions directly in `submissions/`, or two (or more) subdirectories inside it are ignored.

I think this is supposed to mean that all submissions must be in a subdirectory of `submissions/`, no higher and no lower. We should try to write this more clearly.

### PR DESCRIPTION
**Pull request by JoelNiemela**
Closes #306

### CODE REVIEW

**Review by niemela — APPROVED** [review]

File: spec/2023-07-draft.md [hunk 1]
```diff
@@ -689,7 +689,7 @@ that file must have the language-dependent name as given in the table referred t
 
 Correct and incorrect solutions (file or directory programs) to the problem are provided in direct subdirectories of `submissions/`.
 That is By default, the possible subdirectories are as in the table below, but they can be customized, and more can be added; see [Default directories](#default-directories).
-Submissions directly in `submissions/`, or two (or more) subdirectories inside it are ignored.
+Submission programs (either a single file or a directory of files) **must** be placed in a subdirectory of `submissions`, e.g., `submissions/accepted/`.
```
Comment (eldering) [inline spec/2023-07-draft.md:692]:
```suggestion
Submission programs (either a single file or a directory of files) **must** be placed in a subdirectory of `submissions`, e.g., `submissions/accepted/`.
No extra levels of subdirectories are allowed.
```

Comment (eldering) [reply spec/2023-07-draft.md:692]: @JoelNiemela @niemela I think this was not addressed

Comment (JoelNiemela) [reply spec/2023-07-draft.md:692]: @eldering It was not — @niemela seems to have merged this PR while you were reviewing it, and I can't commit the suggestion after it's been merged.

I can create a follow-up PR with this change in a while.

Comment (JoelNiemela) [reply spec/2023-07-draft.md:692]: Done: #319

**Pull request merged**